gravitee-apim-gateway/gravitee-apim-gateway-tests-sdk/src/main/java/io/gravitee/apim/gateway/tests/sdk/AbstractHttp2GatewayTest.java

```diff
@@ -16,22 +16,19 @@
 package io.gravitee.apim.gateway.tests.sdk;
 
 import io.gravitee.apim.gateway.tests.sdk.configuration.GatewayConfigurationBuilder;
-import io.reactiverse.junit5.web.WebClientOptionsInject;
-import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.core.http.HttpClientOptions;
 
 /**
  * @author Yann TAVERNIER (yann.tavernier at graviteesource.com)
  * @author GraviteeSource Team
  */
 public abstract class AbstractHttp2GatewayTest extends AbstractGatewayTest {
 
-    @WebClientOptionsInject
-    public WebClientOptions options = new WebClientOptions()
-        .setDefaultHost("localhost")
-        .setDefaultPort(gatewayPort())
-        .setSsl(true)
-        .setVerifyHost(false)
-        .setTrustAll(true);
+    @Override
+    protected void configureHttpClient(HttpClientOptions options) {
+        options.setDefaultHost("localhost").setDefaultPort(gatewayPort()).setSsl(true).setVerifyHost(false).setTrustAll(true);
+        super.configureHttpClient(options);
```

Comment:
I think you can get rid of it since the implementation is empty

Reply:
done :heavy_check_mark: